src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.scss

```diff
@@ -23,9 +27,14 @@
 }
 
 .geneViewTabs {
-  grid-area: 2 / 2 / 3 / 4;
+  grid-area: 2 / 1 / 3 / 4;
   z-index: 1;
   align-self: end;
```

Comment:
no need for align-self: end then, if the position of the buttons is controlled through position: absolute

Reply:
Removed it